tests/numerics/fv/test_tpsa.py

```diff
@@ -21,1418 +36,1426 @@
 matrices in the dictionary."""
 
 
-def _discretize_get_matrices(grid: pp.Grid, d: dict):
-    """Helper function to discretize with Tpsa and return the dictionary of
-    discretization matrices.
+@pytest.mark.parametrize("g", [pp.CartGrid([2, 2]), pp.CartGrid([2, 2, 2])])
+@pytest.mark.parametrize("driving_bc_type", ["dir", "neu"])
+@pytest.mark.parametrize("tensile", [False, True])
+def test_compression_tension(g: pp.Grid, driving_bc_type: str, tensile: bool):
+    """Assign a compressive or tensile force, check that the resulting displacement is
+    in the correct direction, and that the total pressure is negative.
 
-    Parameters:
-        grid: Grid to discretize.
-        d: Dictionary with parameters.
+    The logic of the test is as follows (assuming a compressive regime, tensile=False,
```

Comment:
Please revisit this paragraph, or risk me sending you this https://xkcd.com/859/ once a day until the parentheses are closed.

Reply:
Please also consider spelling out the type of condition. Both "compressive" and "neutral" could be either dir or neu.